### ISSUE TITLE
Allowing users to set an arbitrary value on text inputs.

### ISSUE THREAD

**Issue by apaganobeleno** [gobuffalo/tags#12]
Sometimes your form receives formatted values for attributes like Date, or does some transformation of the values received from the user before storing into the database.

For those cases it would be useful to allow users to pass a `value` option into the tag that will be the value we'll set into the input.
p.e.:

```erb
<%= f.InputTag("Date", {placeholder:"Date", value: "01-02-2016", label: "Date"}) %>
<%= f.InputTag("Value", {placeholder:"Value", value: "", label: "Value"}) %>
```

@markbates thoughts ?

**Comment by markbates** [comment]
Sounds good to me!

**Comment by apaganobeleno** [comment]
closed by #14
